.rubocop.yml

```diff
@@ -0,0 +1,70 @@
+inherit_gem:
+  jekyll: .rubocop.yml
+
+AllCops:
+  Exclude:
+    - vendor/**/*
+
+Metrics/LineLength:
+  Exclude:
+    - lib/mercenary/command.rb
+    - lib/mercenary/presenter.rb
+    - lib/mercenary/program.rb
+    - examples/**/*
+    - spec/**/*
+    - mercenary.gemspec
+
+Metrics/BlockLength:
+  Exclude:
+    - examples/**/*
+    - spec/**/*
+
+Style/FrozenStringLiteralComment:
+  Enabled: true
+  EnforcedStyle: always
+  Exclude:
+    - lib/mercenary/command.rb
```

Comment:
perhaps we should additionally make this file future-proof by adding `# frozen_string_literal: false`..? Strings are going to be immutable by default around then..